cdc/log.cc

```diff
@@ -1304,6 +1305,15 @@ class transformer final {
                     }
                     return make_lw_shared<cql3::untyped_result_set>(*result_set);
         });
+      } catch (exceptions::unavailable_exception& e) {
```

Comment:
Shouldn't it be a handle_exception on the future returned from _ctx._proxy.query. Otherwise, it is possible for this method to return a non-ready future and the exception you want to catch will be thrown somewhere else, no? Or is the exception thrown from _ctx._proxy.query before creating any future?

Reply:
I thought about it, but this `throw` is buried quite deep in non-deferred code (`storage_proxy::get_read_executor`) so I assumed that throwing is a part of its contract and better not be futurized. I also experimentally tried `handle_exception` here, but it didn't catch anything.